dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/data/upload/UploadWorker.kt

```diff
@@ -51,6 +52,12 @@ internal class UploadWorker(
             TracesFeature.uploader
         )
 
+        // Upload RUM
+        uploadAllBatches(
```

Comment:
what is the impact of not uploading it? I guess we could just miss some batches if phone is completely in background? because as I understand batches also scheduled for the upload in case if app in foreground.